components/SearchSelect/data.js

```diff
@@ -0,0 +1,30 @@
+export const colourOptions = [
```

Comment:
Like I've said before ... add ALL test data to the playground ... no one cares if it's there.

Reply:
Moved this to the playground